packages/plugin-lightningcss/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@rsbuild/plugin-lightningcss",
+  "version": "0.4.14",
+  "description": "lightningcss for Rsbuild",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/web-infra-dev/rsbuild",
+    "directory": "packages/plugin-lightningcss"
+  },
+  "license": "MIT",
+  "type": "commonjs",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "default": "./dist/index.js"
+    },
+    "./loader": {
+      "types": "./dist/loader.d.ts",
+      "default": "./dist/loader.js"
+    },
+    "./minimizer": {
```

Comment:
should `loader` and `minimizer` be a public API? maybe we can just remove these two exports fields.